src/test/java/io/vertx/ext/mail/TestUtils.java

```diff
@@ -69,4 +66,15 @@ public static String inputStreamToString(final InputStream inputStream) throws I
     return string.toString();
   }
 
+  // this method can be deprecated after JDK9+ which has InputStream.readAllBytes()
+  public static byte[] inputStreamToBytes(final InputStream inputStream) throws IOException {
+    ByteArrayOutputStream bufferedArray = new ByteArrayOutputStream();
+    int nRead;
+    byte[] buffer = new byte[4096];
```

Comment:
this seems a bit large, 512 would be better

Reply:
Done by changing to 512